webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/Zarr3BucketStreamSink.scala

```diff
@@ -41,7 +41,7 @@ class Zarr3BucketStreamSink(val layer: VolumeTracingLayer, tracingHasFallbackLay
   def apply(bucketStream: Iterator[(BucketPosition, Array[Byte])], mags: Seq[Vec3Int], voxelSize: Option[VoxelSize])(
       implicit ec: ExecutionContext): Iterator[NamedStream] = {
 
-    val header = Zarr3ArrayHeader.fromDataLayer(layer)
+    val header = Zarr3ArrayHeader.fromDataLayer(layer, mags.head)
```

Comment:
```suggestion
    val header = Zarr3ArrayHeader.fromDataLayer(layer, mags.headOption.getOrElse(Vec3Int.ones))
```
`head` throws exceptions on empty lists. Not sure how to create volume annotations with no mags, but I feel more comfortable with this variant.

Reply:
sure, thanks for the suggestion :pray: